src/io/iologindata.cpp

```diff
@@ -738,6 +692,58 @@ bool IOLoginData::loadPlayer(Player* player, DBResult_ptr result)
   return true;
 }
 
+void IOLoginData::loadRewardItems(Player *player, DBResult_ptr &result) {
```

Comment:
```suggestion
void IOLoginData::loadRewardItems(Player *player, DBResult_ptr &result) {
```
You can migrate this to here: https://github.com/opentibiabr/canary/blob/97f3e8c7c5c8023d48df7b13e9fa531e318966d5/src/io/functions/iologindata_load_player.cpp#L28